### DISCUSSION TITLE
Whether to consider supporting double column menu?

### DISCUSSION THREAD

**Discussion by keepsl** [primefaces/community#105] [PrimeVue Templates]
Hi,
I hope that templates can support dual-column display, that is, when the top horizontal menu is clicked, the left menu is loaded simultaneously. Is it possible to consider it?

**Comment by tugcekucukoglu** [comment]
Hi,

I don't understand exactly what you mean by dual-column. You mean like [MegaMenu](https://primevue.org/megamenu)
